### ISSUE TITLE
Remove Litecoin

### ISSUE THREAD

**Issue by dcousens** [bitcoinjs/bitcoinjs-lib#1089]
As mentioned in https://github.com/bitcoinjs/bitcoinjs-lib/pull/1088#issuecomment-391627886

[coininfo](https://github.com/cryptocoinjs/coininfo) is the recommended way to integrate compatible alt-coins with this library.

**Comment by HashUnlimited** [comment]
As a user I would prefer a more flexible approach where the coin info isn't hard coded in any repo. A cleaner way would be to have only the basics pre-loaded and allow to import from an external source, preferably a .JSON file, which isn't overwritten by npm install. If the file exists, look up the info there, otherwise stick to the basics.

**Comment by rbndg** [comment]
I agree because the library doesn't support the Litecoin testnet anyway.

**Comment by dcousens** [comment]
> A cleaner way would be to have only the basics pre-loaded and allow to import from an external source, preferably a .JSON file

@HashUnlimited `coininfo` is optional,  you can define all your network information in an external JSON file if you'd like.

**Comment by dbolser** [comment]
> I agree because the library doesn't support the Litecoin testnet anyway.

It does now, but not regtest.
